docs/whats-new.md

```diff
@@ -28,12 +28,16 @@ To receive monthly updates on new features and improvements to Tella and stay up
     - Solved visual bug on the report status screen for report upload progress.
 
 
-### iOS: Tella 1.11.0 (X) - Released on September 23, 2024 {#ios-tella-1110-x---released-on-september-23-2024}
+### iOS: Tella 1.11.0 (81) - Released on October 8, 2024 {#ios-tella-1110-81---released-on-october-8-2024}
 - Implemented the Nexcloud server connection that allows Tella users to upload the data they collect directly to either Nexcloud. [Read here more about the connection](/nexcloud).
 - Bug-fixing and small improvements:
-    - To be decided by devs (see the email i sent)
-    
-
+    - Fixed progress view for large files and cancel action in Tella Web reports view.
```

Comment:
maybe we can change it to :
`Fixed progress view and cancel action for large files in the report view.`
because it's not related only for Tella web, it's unified for all the connections

This bug occurred due to the time consumed when deleting metadata.

Reply:
done! thanks :)